lib/backend/syncersv1/updateprocessors/felixnodeprocessor.go

```diff
@@ -145,6 +147,14 @@ func (c *FelixNodeUpdateProcessor) Process(kvp *model.KVPair) ([]*model.KVPair,
 			Value:    vxlanTunlMac,
 			Revision: kvp.Revision,
 		},
+		{
+			Key: model.ResourceKey{
+				Name: name,
+				Kind: "node",
```

Comment:
I think we have constants defined for the Kind in the apis/v3 package. See here: https://github.com/projectcalico/libcalico-go/blob/7be0d765781fd762c2b3f0a3f5392b2b132ba3e3/lib/apis/v3/node.go#L24